### PR TITLE
Update ifttt.markdown

### PR DESCRIPTION
**Pull request by niai**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/ifttt.markdown [hunk 1]
```diff
@@ -34,9 +34,10 @@ You can then consume that information with the following automation:
 ```yaml
 automation:
   trigger:
-    event: ifttt_webhook_received
+    platform: event
```
Comment (arsaboo) [inline source/_components/ifttt.markdown:37]: Did you test this change?

File: source/_components/ifttt.markdown [hunk 2]
```diff
@@ -34,9 +34,10 @@ You can then consume that information with the following automation:
...
+    event_type: ifttt_webhook_received
     event_data:
-      action: call_service
+     action: call_service
```
Comment (arsaboo) [inline source/_components/ifttt.markdown]: Should be two spaces

File: source/_components/ifttt.markdown [hunk 3]
```diff
@@ -34,9 +34,10 @@ You can then consume that information with the following automation:
 ```yaml
 automation:
   trigger:
-    event: ifttt_webhook_received
+    platform: event
+    event_type: ifttt_webhook_received
     event_data:
-      action: call_service
```
Comment (ludeeus) [inline source/_components/ifttt.markdown]: Every indent should be 2 spaces.

From <https://www.home-assistant.io/docs/configuration/yaml/>
_Convention is to use 2 spaces for each level of indentation._

File: source/_components/ifttt.markdown (hunk 1 above)
Comment (niai) [reply source/_components/ifttt.markdown:37]: Yes it wouldn't work the way it was. I tested with sleep as android using.

```
- id: '1525567357516'
  alias: IFTTT
  trigger:
    platform: event
    event_type: ifttt_webhook_received
    event_data:
     action: call_service
  action:
    service_template: "{{ trigger.event.data.service }}"
    data_template:
      entity_id: "{{ trigger.event.data.entity_id }}"
```

**Review by arsaboo — DISMISSED** [review]

**Comment by niai** [comment]
Your right I missed a space, sorry about that it was very late when I sent the pull request. This is my 1st time having to deal with reviewing and changes, I think I approved your change not sure.

trying to go this rout as there have been a few time i thing the document could do with a little change, so I could do with learning all this and I know that the home assistant crew are looking for people to help with the documents.

**Comment by ludeeus** [comment]
@niai You need to push a change to your branch that reverts the removal of that space.
You also need to change the target branch from `rc` to `current` since this is a correction of the current documentation.

**Comment by niai** [comment]
This is a change in the documents for 0.80.0b5, with the new IFTTT integration, its not part of current yet?

**Comment by ludeeus** [comment]
My bad, in that case, I'm not sure which branch it should go to.

File: source/_components/ifttt.markdown (hunk 1 above)
Comment (arsaboo) [reply source/_components/ifttt.markdown:37]: I just tested the existing version and it works fine:
```yaml
- id: ifttt_test
  alias: IFTTT Test
  initial_state: 'on'
  trigger:
    platform: event
    event_type: ifttt_webhook_received
  action:
    - service: logbook.log
      data_template:
        name: "IFTTT event: "
        message: >
          {{trigger.event.data}}
```

**Comment by arsaboo** [comment]
0.80 is already merged....so you can change the branch to `current`

File: source/_components/ifttt.markdown (hunk 1 above)
Comment (rohankapoorcom) [reply source/_components/ifttt.markdown:37]: The existing version doesn't have the `platform: event`, it has `event: ifttt_webhook_received` and does not work as it. This change is needed. See the [event trigger documentation](https://www.home-assistant.io/docs/automation/trigger/#event-trigger).

**Comment by ludeeus** [comment]
This LGTM, @arsaboo any more he/she needs to do?
The current (live) example does not work anymore for users that have upgraded.

**Comment by oncleben31** [comment]
You should merge it quickly. I've lost half an hour to understand how to correct the example promoted with the latest hass.io version and blog post. Let's help the other users.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
I've dismissed the review by @arsaboo and I'll go ahead and approve this one.
We've received multiple PRs for the same thing.

Thanks, @niai! 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**